nodejs/eks/addons/addon.ts

```diff
@@ -24,7 +24,7 @@ import { Cluster } from "../cluster";
 export interface AddonOptions
     extends Omit<aws.eks.AddonArgs, "resolveConflicts" | "clusterName" | "configurationValues"> {
     cluster: Cluster;
-    configurationValues?: object;
+    configurationValues?: pulumi.Input<object>;
```

Comment:
Note: This should be a safe change as the API is currently modeled as an Input already ([ref](https://github.com/pulumi/pulumi-eks/blob/78eeb0b7a55be5ae05138c801b1b689d1f103003/provider/cmd/pulumi-resource-eks/schema.json#L618-L623)) so this isn't a breaking change.

Reply:
Indeed! The types were telling me that an `Input` should be fine but then it put `Calling [toJSON] on an [Output<T>] is not supported` in my JSON blob